certified-connectors/StarRez REST v1/apiDefinition.swagger.json

```diff
@@ -0,0 +1,17799 @@
+                      "value": "Held"
+                    },
+                    {
+                      "displayName": "History",
+                      "value": "History"
+                    },
+                    {
+                      "displayName": "Incident",
+                      "value": "Incident"
+                    },
+                    {
+                      "displayName": "In Room",
+                      "value": "InRoom"
+                    },
+                    {
+                      "displayName": "Master Account",
+                      "value": "MasterAccount"
+                    },
+                    {
+                      "displayName": "Occupant History",
+                      "value": "OccupantHistory"
+                    },
+                    {
+                      "displayName": "Occupant In Room",
+                      "value": "OccupantInRoom"
+                    },
+                    {
+                      "displayName": "Reserved",
+                      "value": "Reserved"
+                    },
+                    {
+                      "displayName": "Tentative",
+                      "value": "Tentative"
+                    }
+                  ]
+                },
+                "TaxExemptionEnum": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Tax Exemption Enum",
+                  "enum": [
+                    "FullExemption",
+                    "None",
+                    "Tax1Exempt",
+                    "Tax2Exempt",
+                    "Tax3Exempt"
+                  ],
+                  "x-ms-enum-values": [
+                    {
+                      "displayName": "Full Exemption",
+                      "value": "FullExemption"
+                    },
+                    {
+                      "displayName": "None",
+                      "value": "None"
+                    },
+                    {
+                      "displayName": "Tax 1 Exempt",
+                      "value": "Tax1Exempt"
+                    },
+                    {
+                      "displayName": "Tax 2 Exempt",
+                      "value": "Tax2Exempt"
+                    },
+                    {
+                      "displayName": "Tax 3 Exempt",
+                      "value": "Tax3Exempt"
+                    }
+                  ]
+                },
+                "Testing": {
+                  "type": "boolean",
+                  "description": "",
+                  "title": "Testing",
+                  "enum": [
+                    true,
+                    false
+                  ]
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/delete/{TableName}.json/{RowID}": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
```

Comment:
@Tony747SREZ empty schema is not allowed, please define a proper schema. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).
Please fix all empty schemas in the swagger file.

Reply:
This is part of our Delete action which should be the only action that has an empty schema. 

The reason we kept it empty is because the schema would vary depending on the input parameters. For example, if deleting from the Entry table, the schema of the deleted Entry record would be returned. If deleting from the Booking table, the schema of the deleted booking record would be returned.

Therefore we were thinking this fell into the special case where the response schema is dynamic.

Reply:
@Tony747SREZ can you please at least define a type for the schema?

Reply:
Sure. I can add object as the type for the schema.